sidebars.js

```diff
@@ -345,6 +345,8 @@ module.exports = {
             "node/cancellation-scopes",
             "node/determinism",
             "node/external-dependencies",
+            "node/failures",
```

Comment:
```suggestion
            "node/failure-classes",
```

Reply:
Went with handling-failure as you suggested